ui/src/app/shared/genericComponents/modal/abstract-modal-line.ts

```diff
@@ -21,6 +21,8 @@ export abstract class AbstractModalLine implements OnInit, OnDestroy, OnChanges
     /** FormGroup ControlName */
     @Input() controlName: string;
 
+    @Input() showIfChannelValueMatchesCondition: number | string | null = null;
```

Comment:
shouldnt this be rather requiredValue or something because the variable itself is not the condition

Reply:
i rename it in value to be matched